### DISCUSSION TITLE
docs.chia.net has launched!

### DISCUSSION THREAD

**Discussion by sargonas** [Chia-Network/chia-blockchain#9605] [Announcements]
We're excited to announce the release of the new Chia Documentation site at https://docs.chia.net!

This is step one in our multi-step process to better overhaul and update the documentation and support we provide to Farmers, Developers, and the rest of the Chia Ecosystem. Full details can be found here: https://www.chia.net/2021/12/17/announcing-our-new-documentation-site.en.html

Additionally, we welcome you to use this comments thread as a place to share feedback and thoughts on Chia’s documentation improvement efforts, so we can better understand where some pain points may be for you.

**Comment by wrestler9595** [comment]
Hello everyone, i don't have very good english so sorry for it.
I reinstalled my chia from 0 because the sync don't work, and when i connect to my wallet i see balance "0" i had 3.5... chia's before reconnection!

**Reply by Gooseboy2234** [reply]
Did you also check that the wallet synced all the blocks, and reports as "Synced"
